packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -65,9 +61,8 @@ const SearchPage = props => {
         <article className={classes.root}>
             <div className={classes.categoryTop}>
                 <div className={classes.totalPages}>
-                    {`${total_count} items`}
+                    {`"${inputText}" - ${total_count} items`}
```

Comment:
Since we delete the search term in the input box after searching (#2150) I humbly propose this little helper for the users.